### ISSUE TITLE
version with libicu63 needed

### ISSUE THREAD

**Issue by arkanoid87** [piksel/phantomjs-raspberrypi#12]

**Comment by piksel** [comment]
Yeah, it's not going to happen anytime soon. It took a couple of days to get it to compile the last time, and the build image I used got destroyed when my apartment got flooded.

**Comment by arkanoid87** [comment]
sorry to hear that, don't worry I think I have a workaround for raspberry people passing by

- apt get install phantomjs  WARNING: loads of dependencies :(
- QT_QPA_PLATFORM=offscreen npm install phantom
- use same environment variable when running node processes

**Comment by piksel** [comment]
You should still be able to run the compiled version with the LD_LIBRARY_PATH env wrapper script:
[phantomjs-armv6-rpi v2.1.1](https://github.com/piksel/phantomjs-raspberrypi/releases/tag/v2.1.1-r)
